### DISCUSSION TITLE
Wallet Connect V2

### DISCUSSION THREAD

**Discussion by BMscis** [reach-sh/reach-lang#1588] [General]
Will there be an update integrating wallet connect V2 ?

**Comment by kcelestinomaria** [comment]
Hi , Wcv2 had some breaking changes, but Reach doesn't seem to have integrated Wcv2 yet. Hope this issue can be attended to so Reach Dapps can seamlessly integrate multiple wallets.

**Comment by BMscis** [comment]
Use txnlab wallet.
https://github.com/TxnLab/use-wallet
